### DISCUSSION TITLE
The diving suit disappears if you order the bot to carry it from the boat to the station!

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14112] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The diving suit disappears from the bot hands
if you order the bot to carry it from the boat to the station!
I suppose this error will also appear in a network game. I haven't checked.

### Reproduction steps

1. start a new single player campaign game.
2. pause the captain outside the boat at the station.
3. pause the engineer inside the boat
4. take the diving suit into the hands of the engineer.
5. switch to captain
6. order the engineer to follow the captain
7. The engineer will come running to the captain with a diving suit in his hands. But when engineer cross intership hatch, the diving suit disappears from his hands.
8. The diving suit has disappeared, it is not on the boat, it is not at the station.
9. If you undock after this, the diving suit will return to the boat.
10. But if you save and restore the game instead of undock, the diving suit will disappear forever.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
In single player without performance fix :BaroDev:

https://github.com/FakeFishGames/Barotrauma/assets/122838333/5b95022f-09b2-42d4-9504-223710cfff63

https://github.com/FakeFishGames/Barotrauma/assets/122838333/8812cfc9-4aea-4710-a6ad-49054d8979b2

**Comment by Regalis11** [comment]
This has been fixed in the latest unstable and the fix is coming up in the next update, closing
